libs/gi/ui/src/components/ad/util.tsx

```diff
@@ -0,0 +1,13 @@
+export function toMSite() {
+  if (
```

Comment:
Not the best function name but who cares (other than me in this exact moment, then I won't care after I click submit)